gui/mozregui/global_prefs.py

```diff
@@ -1,17 +1,17 @@
-from PyQt4.QtCore import QSettings, QVariant
+from PyQt4.QtCore import QVariant
 from PyQt4.QtGui import QDialog
 
 from mozregui.ui.global_prefs import Ui_GlobalPrefs
 from mozregui import patch_requests
 
 from mozregression.network import set_http_session
-
+from mozregression.config import DEFAULT_CONF_FNAME, get_defaults
 
 def get_prefs():
     """
     Return the global prefs as a dict.
     """
-    settings = QSettings()
+    settings = get_defaults(DEFAULT_CONF_FNAME)
```

Comment:
So yeah, this is the right thing to do. But now the settings object is a dict, not anymore a QSettings, so you have to change the next lines.